### PR TITLE
modified footer

### PR DESCRIPTION
**Pull request by abhirangana**
check footer

### CODE REVIEW

File: frontend/src/components/Footer.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.restaurant-footer{
+    font-size: 35px;
+    font-weight: 700;
+    background-color: #205ed1;
+    height: 70px;
```
Comment (sharshach) [inline frontend/src/components/Footer.css:5]: use min-height instead of height

File: frontend/src/components/Footer.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.restaurant-footer{
+    font-size: 35px;
+    font-weight: 700;
+    background-color: #205ed1;
```
Comment (sharshach) [inline frontend/src/components/Footer.css:4]: keep background color same as Top bar #6200ee

**Comment by sharshach** [comment]
Add flex in App.jsx such that footer will come to bottom of screen or after content

image
image


like this

File: frontend/src/components/Footer.css (hunk 2 above)
Comment (abhirangana) [reply frontend/src/components/Footer.css:4]: Done changing

File: frontend/src/components/Footer.jsx [hunk 3]
```diff
@@ -9,8 +9,8 @@ class Footer extends Component {
 
     render() {
         return (
-            <div>
-                <h1>you are in footer</h1>
+            <div className="restaurant-footer">
+                <p className="restaurant-footer-text">Developed with <span className="restaurant-footer-heart"> &hearts;</span> by Team Journey</p>
```
Comment (sharshach) [inline frontend/src/components/Footer.jsx:13]: remove p use div

**Pull request merged**